nncf/experimental/post_training/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -0,0 +1,366 @@
+        me_transformation_layout = TransformationLayout()
+        me_transformation_layout.register(model_extraction_command)
+        extracted_model = model_transformer.transform(me_transformation_layout)
+        return extracted_model
+
+    def _add_statistic_point(self, container: StatisticPointsContainer, point: TargetPoint, axis: int) -> None:
+        """
+        Adds specific statistic point
+
+        :param container: StatisticPointsContainer
+        :param point: TargetPoint for statistic collection
+        :param axis: channel axis for the statistics calculation
+        """
+        stat_collector = self._backend_entity.mean_statistic_collector(reduction_shape=axis,
+                                                                       num_samples=self.number_samples)
+        container.add_statistic_point(StatisticPoint(target_point=point,
+                                                     tensor_collector=stat_collector,
+                                                     algorithm=PostTrainingAlgorithms.FastBiasCorrection))
+
+    def _create_input_data(self,
+                           input_shape: Tuple[int],
+                           input_fp: List[np.ndarray],
+                           input_name: str) -> Dict[str, NNCFTensor]:
+        """
+        Creates input blob for the bias shift calculation
+
+        :param input_shape: input shape for the blob
+        :param input_fp: input data for the blob
+        :param input_name: name for the output dict
+        :return: dictionary of the blob by input name
+        """
+        input_blob = self._backend_entity.create_blob(input_shape, input_fp)
+        input_data = self._backend_entity.nncf_tensor(input_blob)
+        input_data = {input_name: input_data}
+        return input_data
+
+    def _get_bias_shift(self,
+                        engine: Engine,
+                        model: ModelType,
+                        input_blob: Dict[str, NNCFTensor],
+                        channel_axis: Tuple[int],
+                        output_fp: List[np.ndarray],
+                        output_name: str) -> np.ndarray:
+        """
+        Calculates bias shift for the further corretion
+
+        :param engine: backend-specific engine instance for the model execution
+        :param model: backend-specific sub-model for the execution
+        :param input_blob: input data for the execution
+        :param channel_axis: channel axes for the raw data aggregation
+        :param output_fp: output data for the shift calculation
+        :param output_name: name of the output tensor for the data collection
+        :return: calculated bias shift
+        """
+        engine.set_model(model)
+        q_outputs = engine.infer(input_blob)
+        q_outputs = q_outputs[output_name]
+        q_outputs = self._backend_entity.tensor_processor.mean_per_channel(q_outputs, channel_axis).tensor
+        bias_shift = np.array(output_fp) - q_outputs
```

Comment:
It looks like, q_outputs is framework specific tensor thus I'm not sure that the subtraction is defined for any framework.

Reply:
Thank you for the comment about the functional part of PR. Moved the output postprocessing to the `AlgoBackend` entity.